quickstart/2-browser-parallel-computation/package.json

```diff
@@ -50,14 +40,22 @@
         ]
     },
     "devDependencies": {
-        "@fluencelabs/cli": "0.9.1",
-        "@fluencelabs/aqua-lib": "0.7.7",
+        "@fluencelabs/cli": "0.12.1",
```

Comment:
remove

Reply:
isn't 0.12.1 unstable and the agreed upon guidance is to only use stable?